### DISCUSSION TITLE
VitsAudioConfig cannot be imported

### DISCUSSION THREAD

**Discussion by artificialnouveau** [coqui-ai/TTS#1841] [General]
Hi there, 

I'm trying to train a Vits model, I have tried to import the Vits model in google colab in two ways: 

First approach: 
`! pip install TTS==0.7.1`
`import os`

`from trainer import Trainer, TrainerArgs`

`from TTS.tts.configs.shared_configs import BaseDatasetConfig`
`from TTS.tts.configs.vits_config import VitsConfig`
`from TTS.tts.datasets import load_tts_samples`
`from TTS.tts.models.vits import Vits, VitsAudioConfig `
`from TTS.tts.utils.text.tokenizer import TTSTokenizer `
`from TTS.utils.audio import AudioProcessor`

I get the following error:
`ImportError: cannot import name 'VitsAudioConfig' from 'TTS.tts.models.vits' (/usr/local/lib/python3.7/dist-packages/TTS/tts/models/vits.py)`

Second approach:

`! git clone https://github.com/coqui-ai/TTS`
`%cd TTS`
`! pip3 install -e `
`%cd `

I get the following error: 
`ImportError: cannot import name 'VitsAudioConfig' from 'TTS.tts.models.vits' (/usr/local/lib/python3.7/dist-packages/TTS/tts/models/vits.py)`

How can I import the VitsAudioConfig?

**Comment by p0p4k** [comment]
Because `VitsAudioConfig` is not implemented in v0.7.1.
Please download the dev branch.

```
! git clone -b dev https://github.com/coqui-ai/TTS
%cd TTS
! pip3 install -e
%cd
```

**Comment by artificialnouveau** [comment]
Thanks! I realize that now. It works. 

Best Regards,
Ahnjili
